gapii/client/capture.go

```diff
@@ -212,7 +212,10 @@ func (p *Process) Capture(ctx context.Context, start task.Signal, stop task.Sign
 	}
 	go func() {
 		if stop.Wait(ctx) {
-			if err := writeEndTrace(conn); err != nil {
+			if err := writeEndTrace(conn); err == nil {
+				time.Sleep(2 * time.Second)
+				writeErr <- errors.New("traced app is unresponsive")
```

Comment:
Does this mean whenever we do a manual stop, we have a 2seconds delay and this message?
We may do a manual stop for a different reason than an unresponsive app, and in that case it would be best to quickly stop.

Reply:
No, if the app manages to respond with the end trace message within 2 seconds, the main loop and main function will be ended without error and the message will not appear.

Reply:
"Traced Application is unresponsive."
Try to use full sentences in error messages, they show up better in logs.